docs/index.rst

```diff
@@ -13,6 +13,7 @@ What's Here
 
 *   :doc:`specifications` - Contains specification information for the Trossen Arms and related hardware.
 *   :doc:`getting_started` - These guides will walk you through the setup process for your Trossen Arm.
+*   :doc:`tutorials` - Step-by-step guides for using the Trossen Arm with LeRobot for Imitation Learning.
```

Comment:
tutorials will be more general than just lerobot. we'll have ROS 2 and simulations here as well